core/src/classes/plugin.ts

```diff
@@ -134,9 +134,9 @@ export type PluginOptionType =
  * Only returns the nextHighWaterMark and a count of how many records were imported.
  * Within this method you should call plugin.createImport()
  */
-export interface RecordsPluginMethod {
+export interface RecordsPluginMethod<Connection = any> {
   (argument: {
-    connection: any;
+    connection: Connection;
```

Comment:
I like the ability to optionally type `Connection`.  
[Suggestion] to disambiguate this from an Actionhero `Connection`, perhaps `PluginConnection` or `GrouparooPluginConnection` could be the class name?

Reply:
I just discovered that there's already a `PluginConnection` interface, which are the connections you assigned to the plugin. I am thinking about calling this "AppConnection" because it gets generated by the `ConnectPluginAppMethod`